src/test/ui/resolve/issue-14254.stderr

```diff
@@ -2,7 +2,7 @@ error[E0425]: cannot find function `baz` in this scope
   --> $DIR/issue-14254.rs:29:9
    |
 29 |         baz();
-   |         ^^^ did you mean `self.baz(...)`?
+   |         ^^^ help: did you mean `self.baz`
```

Comment:
We could allow a placeholder in the message. So if you write `did you mean $?` it will produce

```
did you mean `foobar`?
```